### ISSUE TITLE
Support "terraform import" for storage pool resources

### ISSUE THREAD

**Issue by nmoun** [NetApp/terraform-provider-netapp-gcp#88]
Problem description:
I am trying to import an existing storage pool into my TF state, using the "terraform import" command. Similar issue than https://github.com/NetApp/terraform-provider-netapp-gcp/issues/33, except it concerns storage pools.

Steps:
1. Create the resource in the TF configuration:
```terraform
resource "netapp-gcp_storage_pool" "default" {
}
```
2. Find the ID of the storage pool to import (e.g. in UI)
3. Run the import
```
terraform import netapp-gcp_storage_pool.default b7d1e1aa-ac21-075b-6e15-06925c49635
```
4. Use terraform show to update the empty resource file with the actual settings.
5. Use terraform plan to verify that the resource file against the deployed storage pool. 

Problem: the import fails:
```
terraform import "netapp-gcp_storage_pool.default" "b7d1e1aa-ac21-075b-6e15-06925c49635"
netapp-gcp_storage_pool.default: Importing from ID "b7d1e1aa-ac21-075b-6e15-06925c49635"...
netapp-gcp_storage_pool.default: Import prepared!
  Prepared netapp-gcp_storage_pool for import
netapp-gcp_storage_pool.default: Refreshing state... [id=b7d1e1aa-ac21-075b-6e15-06925c49635]
╷
│ Error: <html><body><h1>403 Forbidden</h1>
│ Request forbidden by administrative rules.
│ </body></html>
```

Reason: the region must be provided.

**Comment by okrause** [comment]
Acknowledged. The volume code is currently the only resource which supports importing.
As a reference: Adding import support for other resources would be similar to the code in [getVolumeByID](https://github.com/NetApp/terraform-provider-netapp-gcp/blob/44143d1e45f3c389fa23d8bf4249b863c705d437/gcp/volume.go#L162).

**Comment by wenjun666** [comment]
fix in release 23.4.0
